docusaurus/docs/operate/configs/relayminer_config.md

```diff
@@ -451,3 +457,44 @@ flowchart
 ## Supported server types
 
 The list of supported server types can be found at [pkg/relayer/config/types.go](https://github.com/pokt-network/poktroll/tree/main/pkg/relayer/config/types.go#L8)
+
+## Payable Proof Submissions
+
+### Overview
+
+Submitting `Proofs` requires the payment of a fee, which is defined by the
+`proof_submission_fee` governance parameter.
+
+This fee mechanism is designed to prevent spam attacks (e.g., Sybil bloat attacks)
+by discouraging invalid or unnecessary `Proofs`.
+
+Since `Proofs` are the most block space-consuming primitive in the Pocket Network,
+this fee plays a crucial role in maintaining the integrity of the network.
+
+### Key Requirements for Operators
+
+Due to the probabilistic nature of the `Proof` requirement, `Supplier` operators must
+ensure that they maintain sufficient funds to cover the submission fees for all
+`Proof`s associated with the `Claim`s they plan to submit.
+
+Failure to maintain adequate funds can result in missed submissions, which can
+result in `Supplier` slashing if the `Proof` is required.
+
+### Recommendations for Supplier Operators
+
+- **Sufficient Balance**: Operators should regularly check their account balance
```

Comment:
#PUD (Please update document) w/ 

1. A copy-pasta command to check your balance
2. A copy pasta command to check fee for a proof.